src/SetUnits.jl

```diff
@@ -1,47 +1,57 @@
-
 # include("PhysicalConstants.jl")
```

Comment:
Includes should be suppressed in the release version, as repeated includes cause errors and everything is included directly in AtomicAndPhysicalConstants.jl